docs/api/adaptors/uniswap/uniswap_v3.md

```diff
@@ -7,6 +7,7 @@
     - [UniswapV3Adaptor](#steward-v3-UniswapV3Adaptor)
     - [UniswapV3Adaptor.AddToPosition](#steward-v3-UniswapV3Adaptor-AddToPosition)
     - [UniswapV3Adaptor.ClosePosition](#steward-v3-UniswapV3Adaptor-ClosePosition)
+    - [UniswapV3Adaptor.CollectFees](#steward-v3-UniswapV3Adaptor-CollectFees)
```

Comment:
What's going on here? Did we forget to regenerate the docs when porting forward from 2.x?

Reply:
yes